#!/usr/bin/env python
PACKAGE="vn_100"
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()
gen.add("kp_pitch",double_t,0,"pitch proportional param",0,0,300)
gen.add("ki_pitch",double_t,0,"pitch integral parameter",0,0,300)
gen.add("kd_pitch",double_t,0,"pitch derivative parameter",0,0,300)
gen.add("setpoint",double_t,0,"pitch setpoint",0,-90,90)
gen.add("best_error",double_t,0,"best error value",0,-100,100)
gen.add("bool_param",bool_t,0,"to time the values",False)
gen.add("bool_twiddle",bool_t,0,"to give the best error vale",False)
exit(gen.generate(PACKAGE,"control","pitchparams"))